volatility3/framework/plugins/windows/handles.py

```diff
@@ -135,78 +99,6 @@ def _get_item(self, handle_table_entry, handle_value):
         object_header.HandleValue = handle_value
         return object_header
 
-    def find_sar_value(self):
```

Comment:
This method doesn't start with an `_`.  It is an instance method, so there shouldn't be anyone calling this externally (and this is why I'd prefer things that are supposed to be internal only to be prepended with an `_`), but I really want to try and do things correctly rather than messing things up and getting a reputation as being unreliable because we made corner cases for our own rules.

As such, can this please be turned into a MAJOR version bump (rather than just a PATCH bump) due to the removal of this method, and all plugins that depend on it be updated to depend on the newer version (I think that's just `poolscanner`, `psxview`, `dumpfiles` and `callbacks` plugins).  Either that, or restore this deleted method (perhaps with a `DeprecationWarning` at the start?

Reply:
Major version bumped on plugin + dependents 👍